### PR TITLE
Add Electron Builder icon

### LINKED ISSUE #6115: Request: electron-builder icon
**Brand Name:** electron-builder
**Website:** https://www.electron.build/
**Alexa rank:** 133,332
  

**Official resources for icon and color:** https://github.com/electron-userland/electron-builder

### PR DESCRIPTION
**Pull request by PaulaBarszcz**
electronbuilder_updated

**Issue:** closes #6115 
There was a PR opened before but it was closed without merge - https://github.com/simple-icons/simple-icons/pull/6177

**Similarweb rank:** 528,375

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

  - Why did you pick the hex value?
I chose  #2094F3 as a primary colour - it is the colour of the header on https://www.electron.build/ 
image

  - Did you manually vectorize the logo?
  No, I used an SVG version of the logo from https://www.electron.build/, 
  resized it in Inkscape so that 2 sides touch the border,
  centred horizontally & vertically,
  added `title` and `role`,
  and ran SVGO.
  
  - Have you used multiple sources?
  No.

### CODE REVIEW

**Comment by PaulaBarszcz** [comment]
I see that running 'Minify SVG' in VSCode removed some of its properties (role and title), I will add them ;)

**Comment by PaulaBarszcz** [comment]
I see that I will have to slightly modify the icon's dimensions in Inkscape - I will do that when I have some time :)

**Comment by PaulaBarszcz** [comment]
Hi @sisp,
When you have time, could you please take a look and let me know if anything is missing in this PR for it to be merge-ready for the develop branch?
If everything seems ok, it would be great if you could merge it (or hint as to who might do it instead) :)
Thanks a lot!

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3184,6 +3184,11 @@
             "hex": "47848F",
             "source": "https://www.electronjs.org/"
         },
+        {
+            "title": "Electron Builder",
```
Comment (sisp) [inline _data/simple-icons.json:3188]: It seems the official spelling is "electron-builder".

```suggestion
            "title": "electron-builder",
```

**Review by sisp — CHANGES_REQUESTED** [review]
Thanks for your contribution, @PaulaBarszcz! :pray:

I see no notable difference between your icon and the original one :+1::


I left some comments though.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -3184,6 +3184,11 @@
             "hex": "47848F",
             "source": "https://www.electronjs.org/"
         },
+        {
+            "title": "electron-builder",
+            "hex": "2094F3",
```
Comment (PaulaBarszcz) [inline _data/simple-icons.json:3189]:
```suggestion
            "hex": "FFFFFF",
```

File: _data/simple-icons.json [hunk 3]
```diff
@@ -3184,6 +3184,11 @@
             "hex": "47848F",
             "source": "https://www.electronjs.org/"
         },
+        {
+            "title": "Electron Builder",
+            "hex": "2094F3",
```
[the remark this answers is not in the document]
Comment (sachinraja) [reply _data/simple-icons.json]: hello, I agree, the brand color is white

**Review by sisp — APPROVED** [review]
LGTM now! :tada: Thank you! :pray:

**Pull request merged**